### PR TITLE
Adds missing module to viz environment

### PR DESCRIPTION
**Pull request by NadirRoGue**

### CODE REVIEW

**Review by matz-e — COMMENTED** [review]
I don't see the module in the list to test, don't you need to add it below to the `specs:` key, too?

**Comment by NadirRoGue** [comment]
Actually, I added it on the `specs:` and forgot to add it on the `modules:` . Thanks for spotting it!

**Comment by NadirRoGue** [comment]
@matz-e Where should I add py-websockets so that it can be used for py-brayns?

File: bluebrain/deployment/environments/applications_libraries.yaml [hunk 1]
```diff
@@ -25,6 +25,7 @@ spack:
         - py-scikit-learn
         - py-scipy
         - py-tensorflow
+        - py-websockets
```
Comment (matz-e) [inline bluebrain/deployment/environments/applications_libraries.yaml]: Both these modifications should disappear again, no changes to `libraries` needed.

I overlooked it, but `py-websockets` needs to be a dependency with `type=("build", "run")`, as that should be always done for Python dependencies.

Comment (NadirRoGue) [reply bluebrain/deployment/environments/applications_libraries.yaml]: Thanks! I applied these changes.

**Review by matz-e — APPROVED** [review]

**Pull request merged**